eurec4a_environment/variables/boundary_layer/mixed_layer_height.py

```diff
@@ -0,0 +1,22 @@
+import xarray as xr
+
+
+def calc_peak_RH(ds, altitude="alt", rh="RH", z_min=200.0, z_max=900.0):
+    """
+    Calculate height at maximum relative humidity values
+    """
+    ds_mixed_layer = ds.sel({altitude: slice(z_min, z_max)})
+
+    da_rh = ds_mixed_layer[rh]
+    dims = list(ds.dims.keys())
+    # drop the height coord
+    del dims[dims.index(altitude)]
+
+    peakRH_idx = da_rh.argmax(dim=altitude)
+    h_peakRH = ds[altitude].isel({altitude: peakRH_idx})
```

Comment:
Do you have to add index of z_min=20 to peakRH_idx? For the results, I see heights that range from 20-700m

Reply:
You're right that the indexing was wrong. I was picking by number into an array where I'd already sliced off values above and below 😄